sdk/test-utils/recorder/package.json

```diff
@@ -63,17 +63,24 @@
     "fs-extra": "^8.1.0",
     "nise": "^1.4.10",
     "nock": "^11.7.0",
-    "tslib": "^1.10.0"
+    "tslib": "^1.10.0",
+    "md5": "~2.2.1",
```

Comment:
should these be `^` deps like the above?

Reply:
Sure! though, why? I'm not sure if this will add any benefit. I'd like to learn!

Reply:
Doing it!